website/docs/getting-started/setup/instance-configuration/custom-mongodb-redis.md

```diff
@@ -1,38 +1,83 @@
 ---
 sidebar_position: 3
+toc_max_heading_level: 4
 ---
-# Custom MongoDB & Redis
+# External MongoDB and Redis
 
-Appsmith runs it's own instances of MongoDB database for storing all application data, and Redis for storing session information. It's easy to switch to externally running instances of these services, by just changing the relevant environment variables.
+Appsmith runs embedded instances of MongoDB and Redis. Appsmith uses MongoDB and Redis for data storage and session management. You can use external instances by configuring the instance. External instances are stateless and help in better performance and scalability. Follow the steps on this page to set up external MongoDB and Redis for Appsmith.
 
-:::info Important
-If you are currently using an older version of MongoDB, it's important to upgrade to version 5.0 before connecting Appsmith to an external MongoDB instance. For more information, follow the steps detailed on MongoDB official documentation to [Upgrade a Replica Set to 5.0](https://www.mongodb.com/docs/manual/release-notes/5.0-upgrade-replica-set/).
+:::caution Attention
+To use a custom MongoDB with Appsmith v1.9.0 and later, you need MongoDB version 5 or higher. Follow the steps detailed in the MongoDB official documentation to [Upgrade a Replica Set to 5.0](https://www.mongodb.com/docs/manual/release-notes/5.0-upgrade-replica-set/).
 :::
 
-## Custom MongoDB
+## External MongoDB
+Follow the below steps to set up an external MongoDB:
 
-To point to a custom MongoDB, update the `env` variable called `APPSMITH_MONGODB_URI` to point to your custom MongoDB server. For example, if you are using [MongoDB Cloud](https://www.mongodb.com/cloud), you can change this to something like the following:
+### Prerequisites
+* Ensure that your external MongoDB has a replica set configuration in place. Connect to your database as an admin user and run [rs.initiate()](https://docs.mongodb.com/manual/reference/method/rs.initiate/). Note that if you're using MongoDB Cloud, the replica set configuration is already set up for you.
+* Ensure that the user you are connecting with has `readWrite` and `clusterMonitor` permissions.
```

Comment:
It looks a bit confusing.  How about? 

```suggestion
* Ensure the MongoDB user account has `readWrite` and `clusterMonitor` roles assigned.

Reply:
Rewriting this as - Ensure that the user you are connecting with has `readWrite` and `clusterMonitor` roles assigned. ( as we are in MongoDB section, so IMO we shouldn't add MongoDB to specify that)